src/index.js

```diff
@@ -37,6 +38,9 @@ ReactDOM.render(
           <Route path="add-event" component={Overview} >
             <IndexRoute component={AddEvent} />
           </Route>
+          <Route path="profile" component={Overview} >
```

Comment:
Do you now want the map to be behind the profile?
Otherwise modify it to what we talked about.

Instead of 
```
<Route path="profile" component={Overview}>
   <IndexRoute ..... />
</Route>
```

only 
```
<Route path="profile" component={Profile} />
```